<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-HTHHALI" name="com.salesforce.testcases.TC003_DashboardAssessment" tests="2" failures="0" timestamp="2022-12-14T22:43:50 IST" time="289.187" errors="1">
  <testcase name="createDashboard" time="112.912" classname="com.salesforce.testcases.TC003_DashboardAssessment"/>
  <system-out/>
  <testcase name="createDashboard" time="176.275" classname="com.salesforce.testcases.TC003_DashboardAssessment">
    <error type="java.lang.RuntimeException" message="See the reporter for details.">
      <![CDATA[java.lang.RuntimeException: See the reporter for details.
at com.framework.utils.Reporter.reportStep(Reporter.java:93)
at com.framework.utils.Reporter.reportStep(Reporter.java:108)
at com.framework.selenium.api.base.SeleniumBase.locateElement(SeleniumBase.java:620)
at com.salesforce.pages.DashboardPage.enterDashboardName(DashboardPage.java:23)
at com.salesforce.testcases.TC003_DashboardAssessment.createDashboard(TC003_DashboardAssessment.java:40)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- createDashboard -->
  <system-out/>
</testsuite> <!-- com.salesforce.testcases.TC003_DashboardAssessment -->
